circle.yml

```diff
@@ -691,7 +691,7 @@ jobs:
           command: |
             export RUSTFLAGS="-Z sanitizer=address -C opt-level=0"
             export RUSTDOCFLAGS=$RUSTFLAGS
-            cargo +nightly test --target x86_64-unknown-linux-gnu
+            ASAN_OPTIONS=detect_leaks=1 cargo +nightly test --target x86_64-unknown-linux-gnu
```

Comment:
why not use `export` as in above lines?

Reply:
Moved this around, still works as expected: https://app.circleci.com/pipelines/github/wasmx/fizzy/6906/workflows/e190b481-3c7e-4cf6-a7cd-15f26b69eb7b/jobs/62392